src/main/java/io/pakland/mdas/githubstats/application/internal/CheckMetricsFromUserByRange.java

```diff
@@ -0,0 +1,39 @@
+package io.pakland.mdas.githubstats.application.internal;
```

Comment:
Yes, of course, my branch was updated to the latest main. I have checked now what happened and seems a side effect because my changes were older than the latest main changes that we agreed on at the meeting. So I made a rebase, and it maintained the internal structured, and didn't notice that. My mistake

![image](https://user-images.githubusercontent.com/117458964/210116518-1c9894ee-23d0-496c-9b78-ad4f7d68ba3b.png)

Reply:
> We have reviewed this pr together (@manerow and I). We believe that some things that have been done were a bit out of the scope of the ticket. If we have created smaller tickets for the tasks that have been done in this ticket, doing a rollback would have been easier. Now, it is harder to revert changes, as the ones we are requesting.

Yeah, that is true, this PR was huge and I was sorry about that. But the motivation was that we were in hurry, and many pieces weren't working together yet. I tried my best to address all issues that blocked my history deployment. Others were outside the scope as you commented.